### PR TITLE
Remove useless recipes

### PR DESCRIPTION
**Pull request by Prometheus0000**
See https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7602
They just clutter up NEI, increase loading time, and no-one uses them.
More removals needed in GT5U.

This doesn't actually include any chemical reactor recipe changes/additions though.

### CODE REVIEW

**Comment by Dream-Master** [comment]
Not sure if they are uselss. Is there an rfc issue about it or the issue you linked ?

**Comment by Prometheus0000** [comment]
They are useless. They're just recipes where you can use small or tiny dusts instead of full. But as I said, no-one would go to the effort of splitting the full dusts to make them smaller to do this.

**Comment by Prometheus0000** [comment]
image
image
image
This kind of thing

**Comment by boubou19** [comment]
i agree, those recipes should not exist, as dealing with tiny piles is a true pain.

**Comment by mitchej123** [comment]
I agree, the mixer recipes for the smaller dusts are silly and nobody uses them

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java [hunk 1]
```diff
@@ -3562,7 +3218,7 @@ public void run3() {
         GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_ModHandler.getModItem("IC2","reactorReflector",1L,1), GT_ModHandler.getModItem("IC2","reactorReflector",1L,1), GT_ModHandler.getModItem("IC2","reactorReflector",1L,1), GT_ModHandler.getModItem("IC2","reactorReflector",1L,1), GT_ModHandler.getModItem("IC2","reactorReflector",1L,1), GT_Utility.getIntegratedCircuit(5)}, GT_Values.NF, CustomItemList.NeutronReflectorSmallParts.get(1), 300, 120);
         GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_ModHandler.getModItem("IC2","reactorReflectorThick",1L,0), GT_ModHandler.getModItem("IC2","reactorReflectorThick",1L,0), GT_ModHandler.getModItem("IC2","reactorReflectorThick",1L,0), GT_ModHandler.getModItem("IC2","reactorReflectorThick",1L,0), GT_ModHandler.getModItem("IC2","reactorReflectorThick",1L,0), GT_ModHandler.getModItem("IC2","reactorReflectorThick",1L,0), GT_ModHandler.getModItem("IC2","reactorReflectorThick",1L,0), GT_ModHandler.getModItem("IC2","reactorReflectorThick",1L,0), GT_Utility.getIntegratedCircuit(8)}, GT_Values.NF, CustomItemList.NeutronReflectorParts.get(1), 600, 120);
 
-        //Arcane Slaps -> Arcane Stone
+        //Arcane Slabs -> Arcane Stone
```
Comment (mitchej123) [inline src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:3221]: wtf do you have against arcane slaps

**Review by mitchej123 — APPROVED** [review]

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java [hunk 2]
```diff
@@ -720,10 +721,9 @@ public void run() {
...
         GT_Values.RA.addElectrolyzerRecipe(ItemList.Cell_Empty.get(1), GT_ModHandler.getModItem("TwilightForest", "tile.AuroraSlab", 2L, 0), GT_Values.NF, Materials.Helium_3.getFluid(50), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Stone, 1L), GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Helium, 1), GT_Values.NI, GT_Values.NI, GT_Values.NI, GT_Values.NI, null, 600, 480);
         GT_Values.RA.addElectrolyzerRecipe(ItemList.Cell_Empty.get(1), GT_ModHandler.getModItem("TwilightForest", "tile.AuroraDoubleSlab", 1L, 0), GT_Values.NF, Materials.Helium_3.getFluid(100), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Stone, 1L), GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Helium, 1), GT_Values.NI, GT_Values.NI, GT_Values.NI, GT_Values.NI, null, 600, 480);
-        if (Loader.isModLoaded("gendustry")) {
-
-            GT_Values.RA.addCentrifugeRecipe(CustomItemList.FieryBloodDrop.get(1L), GT_Values.NI, GT_Values.NF, Materials.FierySteel.getFluid(10L), GT_Values.NI, GT_Values.NI, GT_Values.NI, GT_Values.NI, GT_Values.NI, GT_Values.NI, new int[]{10000}, 12, 480);
-        }
+        if (Loader.isModLoaded("gendustry"))
```
Comment (boubou19) [inline src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:724]: Any idea why this is added only if gendustry is present? I mean, the fiery blood drop is an item from the core mod, it comes from the hydra comb, and iirc all the gendustry bees are now GT bees.

Comment (Prometheus0000) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:724]: It should probably be TF, but I didn't have it loaded to check.

**Comment by mitchej123** [comment]
It's removing bloat.  Can you find one person who has ever used these recipes?

**Comment by interestingusernam3** [comment]
@0lafe

**Comment by GTNH-Colen** [comment]
5th'd (?) I don't see why these recipes need to exist, cut the bloat.

**Comment by 0lafe** [comment]
> @0lafe

yes 

This seems very irrelevant, no? Has anyone ebf'd a non full dust?

**Comment by Glease** [comment]
There is only one hypothetical use case: you have a trace amount of some material that you can't produce and you want to make some alloy off it. AFAIK this situation cannot happen in GTNH unless your base has just been blown up by wither/overvolt/creeper/fire.

**Comment by interestingusernam3** [comment]
>There is only one hypothetical use case: you have a trace amount of some material that you can't produce and you want to make some alloy off it. AFAIK this situation cannot happen in GTNH unless your base has just been blown up by wither/overvolt/creeper/fire.

You can also obtain trace amounts of materials from chest loot and very slow processes (other options exist for dealing with those tho, like parallelization), although I'm not aware of any specific material that can be obtained in those ways and for which use of those recipes makes any sense at all.

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**